tests/zfs-tests/tests/functional/cli_root/zfs_unshare/zfs_unshare_006_pos.ksh

```diff
@@ -44,9 +44,9 @@ verify_runnable "global"
 function cleanup
 {
 	log_must $ZFS unshare -a
+	log_must $ZFS destroy -f $TESTPOOL/$TESTFS/shared
 	log_must $ZFS destroy -f $TESTPOOL/$TESTFS/shared1
 	log_must $ZFS destroy -f $TESTPOOL/$TESTFS/shared2
-	log_must $ZFS destroy -f $TESTPOOL/$TESTFS/shared3
```

Comment:
I had inadvertently used `$TESTDIR1, $TESTDIR2, $TESTDIR3` instead of `$TESTDIR $TESTDIR1, $TESTDIR2`, i changed also the datasets names to keep things consistent but forgot to replace all occurrences.